src/Components/Common/Input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {ChangeEvent} from "react";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
+  color: #c2c0c0;
+  background-color: #2f2f2f;
+  border-radius: 2px;
+`;
+
+interface Props {
+    type: string;
+    placeholder: string;
+    width: string;
+    height: string;
+    handleInputChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const Input = (props: Props) => (
+    <StyledInput
+        type={props.type}
+        style={{width: props.width, height: props.height}}
```

Comment:
Zmienne w styled components wstrzykuje się w trochę inny sposób, tak żeby nie były inline kiedy jest to konieczne:
const StyledInput = styled.input`
    width: ${props => props.width};
    height: ${props => props.height};
`
return <StyledInput width:="100px" height="20px"/>

Reply:
No właśnie poczytałem sobie o tym, spoko to przerabiam!